ios/FluentUI/Resources/FluentUI-ios.xcassets/dismiss-20x20.imageset/Contents.json

```diff
@@ -1,15 +1,16 @@
 {
-  "images" : [
+  "images": [
     {
-      "idiom" : "universal",
-      "filename" : "ic_fluent_dismiss_20_regular.pdf"
+      "filename": "ic_fluent_dismiss_20_filled.pdf",
```

Comment:
In the relatively near term are we looking at just taking a dependency on the icons repo rather than just copying icons in?

Reply:
Yes. This is definitely a improvement worth pursuing.